src/main/java/connectify/model/ModelManager.java

```diff
@@ -42,6 +42,8 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
         logger.fine("Initializing with address book: " + addressBook + " and user prefs " + userPrefs);
 
         this.addressBook = new AddressBook(addressBook);
+        // Add a default company to the address book - To be implemented in future
```

Comment:
Let's create an issue for this or take note under our weekly agenda

Reply:
Agreed. Let's create an issue for this under Milestone v1.3.